packages/medusa/src/api/routes/admin/order-edits/delete-order-edit.ts

```diff
@@ -0,0 +1,71 @@
+import { EntityManager } from "typeorm"
+import { OrderEditService } from "../../../../services/order-edit"
+
+/**
+ * @oas [delete] /order-edits/{edit_id}
+ * operationId: "DeleteOrderEditsOrderEdit"
+ * summary: "Delete an Invite"
+ * description: "Deletes an Order Edit"
+ * x-authenticated: true
+ * parameters:
+ *   - (path) edit_id =* {string} The ID of the Order Edit
```

Comment:
suggestion: depends on previous resolution `id` instead

Reply:
I ll let you depending on the resolution, update it everywhere including in the comments and oas